fractal-puppy/Engine
# Bug/Enemies hit box

Bugs fixed:
- [#1193](https://app.hacknplan.com/p/85404/kanban?categoryId=0&boardId=265751&taskId=1193&tabId=basicinfo) Enemies hitbox was activated before the hit animation.
- [#1194](https://app.hacknplan.com/p/85404/kanban?categoryId=0&boardId=265751&taskId=1194&tabId=basicinfo) Elite enemies had a the hitbox to high, so they couldn't hurt the player.

To test:
- Go against an enemy to check that the damage is made with good timing.
- Go against an elite basic enemy and check that it can hurt you.
